packages/cfd/src/Stores/Modules/CFD/Helpers/useCfdStores.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { createContext, PropsWithChildren, useContext } from 'react';
+import { useStore } from '@deriv/stores';
+import type { TCFDStore } from '../../../../types/cfd-store.types';
+
+const CFDStoreContext = createContext<TCFDStore | null>(null);
+
+export const CFDStoreProvider = ({ children }: PropsWithChildren<unknown>) => {
```

Comment:
```suggestion
export const CFDStoreProvider = ({ children }: { children: React.ReactNode }) => {
```

Reply:
@niloofar-deriv Why? :/

Reply:
we just have children here, it's kinda weird to pass unknown here

Reply:
@niloofar-deriv It's better to use React generics as something might change in the future in their API so we don't have to make changes 🙇🏻

Reply:
That's fine, @amir-deriv you can ignore this one